nibabel/gifti/tests/test_gifti.py

```diff
@@ -11,8 +11,7 @@
 from nibabel.gifti.gifti import data_tag
 from nibabel.nifti1 import data_type_codes, intent_codes
 
-from numpy.testing import (assert_array_almost_equal,
-                           assert_array_equal)
+from numpy.testing import (assert_array_almost_equal, assert_array_equal)
```

Comment:
Drop the parentheses.

The commit message makes bc017aa0713193969ecffcca2c7a82a421fbe1e8 look like it must have other stuff rebased out. Maybe re-write that one to match what's left?